R/tcplObj.R

```diff
@@ -49,6 +49,8 @@ tcplObj = function(p, conc, resp, fname, errfun = "dt4", err = NULL) {
 
 #' Constant Model - returns zero
```

Comment:
One last polishing note.  I would remove the "returns zero" here and just have "Constant Model" in the title.